cypress/fixtures/accountTypes.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Should be constants not a JSON ;) 
ex

```
export const PLANS = {
  free: 'free'
} 
``` etc.

But for this one I don't really understand what it does.